impeller/aiks/aiks_unittests.cc

```diff
@@ -503,6 +503,25 @@ TEST_P(AiksTest, CanPictureConvertToImage) {
   ASSERT_TRUE(OpenPlaygroundHere(canvas.EndRecordingAsPicture()));
 }
 
+TEST_P(AiksTest, CanEmptyPictureConvertToImage) {
```

Comment:
Please add a link to this bug in a comment.  It's a weird enough test that it's intent isn't obvious.

Reply:
Done